src/components/ContentNode/TabNavigator.vue

```diff
@@ -71,6 +71,22 @@ export default {
       currentTitle: this.titles[0],
     };
   },
+  watch: {
+    titles(newVal, oldVal) {
+      if (newVal.length < oldVal.length) {
+        // set to the first tab if selected tab was removed
+        const removedTab = oldVal.find(tab => !newVal.includes(tab));
+        if (removedTab === this.currentTitle) {
+          // eslint-disable-next-line prefer-destructuring
+          this.currentTitle = newVal[0];
```

Comment:
Do we want to reset it to the first one or would it make more sense to select the one immediately before/after the one that was removed?

Reply:
We are keeping whatever tab is selected at the time of deletion. If the deleted tab is the selected tab, we just set it to the first item.